eventdiscovery-sdk/src/main/java/com/schedjoules/eventdiscovery/eventlist/itemsprovider/EventListItemsProviderImpl.java

```diff
@@ -102,6 +98,8 @@ public EventListItemsProviderImpl(FutureServiceConnection<ApiService> apiService
         mIsLoading = new HashMap<>(2);
         mIsLoading.put(TOP, false);
         mIsLoading.put(BOTTOM, false);
```

Comment:
This line is when the HashMap gets reashed. Btw. there is an EnumMap in Java that optimized for this case (because the maximum number of keys is known and they don't need to be hashed).